WordPress/Classes/ViewRelated/Notifications/Controllers/NotificationsViewController.m

```diff
@@ -1,258 +0,0 @@
-#import "NotificationsViewController+Internal.h"
-
-#import <Simperium/Simperium.h>
-#import "WordPressAppDelegate.h"
-#import "ContextManager.h"
-#import "Constants.h"
-#import "WPGUIConstants.h"
-
-#import "WPTableViewHandler.h"
-#import "WPWebViewController.h"
-#import "WPTabBarController.h"
-
-#import "Notification.h"
-#import "Meta.h"
-
-#import "NotificationDetailsViewController.h"
-
-#import "WPAccount.h"
-
-#import "AccountService.h"
-
-#import "ReaderPost.h"
-#import "ReaderPostService.h"
-
-#import "UIView+Subviews.h"
-
-#import "WordPress-Swift.h"
-
-
-
-#pragma mark ====================================================================================
-#pragma mark Constants
-#pragma mark ====================================================================================
-
-static CGFloat const NoteEstimatedHeight = 70;
-
-
-
-#pragma mark ====================================================================================
-#pragma mark NotificationsViewController
-#pragma mark ====================================================================================
-
-@implementation NotificationsViewController
-
-- (void)dealloc
-{
-    [[NSNotificationCenter defaultCenter] removeObserver:self];
-}
-
-- (instancetype)initWithCoder:(NSCoder *)aDecoder
```

Comment:
I don't see initWithCoder in the new Swift version. I see bits and pieces of this in the new `viewWillAppear`, but it's not necessarily the same. Can you guarantee the functionality is the same?